ifopt_ipopt/src/ipopt_adapter.cc

```diff
@@ -107,17 +112,29 @@ bool IpoptAdapter::eval_jac_g(Index n, const double* x, bool new_x,
                               Index m, Index nele_jac, Index* iRow, Index *jCol,
                               double* values)
 {
-  // defines the positions of the nonzero elements of the jacobian
```

Comment:
I think the above comment is more precise, let's keep it for now.

Reply:
Done